src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -642,51 +670,115 @@ def create_service_principal_for_rbac(
     return result
 
 
-def _create_self_signed_cert(years):
+def _get_keyvault_client():
+    from azure.cli.core._profile import Profile
+    from azure.keyvault import KeyVaultClient, KeyVaultAuthentication
+
+    def _get_token(server, resource, scope):  # pylint: disable=unused-argument
```

Comment:
It would, however I leave the variable names there in case, in the future, someone needs to make use of these.  Otherwise they wouldn't have any idea what the two _ are supposed to be.